.renovaterc.json5

```diff
@@ -46,7 +46,7 @@
     "VUIIS-dax": {
       defaultRegistryUrlTemplate: "https://api.github.com/repos/VUIIS/dax/contents/misc/xnat-plugins",
       transformTemplates: [
-        '{"releases": $.{"version": $join(["v", $match(name, /(\\d+\\.)?(\\d+\\.)?(\\*|\\d+)/ ).match ])}}',
+        '{"releases": $.{"version": $match(name, /(\\d+\\.)?(\\d+\\.)?(\\*|\\d+)/ ).match }}',
```

Comment:
removing the `v` in the version name (i.e. `1.4.2` rather than `v1.4.1`) to match the format of `currentValue`